spec/javascripts/actions/projectBoard_spec.js

```diff
@@ -1,30 +1,92 @@
-import { expandStoryIfNeeded } from '../../../app/assets/javascripts/actions/projectBoard';
-import { toggleStory } from '../../../app/assets/javascripts/actions/story';
+import { 
+  expandStoryIfNeeded, closeSearch, 
+  closeSearchSuccess, sendErrorNotificationIfNeeded
+} from '../../../app/assets/javascripts/actions/projectBoard';
+import { toggleStory, receiveStories } from '../../../app/assets/javascripts/actions/story';
+import { addNotification } from '../../../app/assets/javascripts/actions/notifications';
 
 describe('Project Board Actions', () => {
-  it('Should dispatch toggleStory when storyId is true', async () => {
-    const storyId = 127;
+  describe('expandStoryIfNeeded', () => {
+    describe('when storyId is true', () => {
+      it('Should dispatch toggleStory', () => {
```

Comment:
You can remove the word `should` here ;)

should definition:
> used to indicate what is probable.

I beilieve is not _probable_ that `dispatch toggleStory` ;)